bus/rabbitmq/bus.js

```diff
@@ -140,31 +149,35 @@ RabbitMQBus.prototype.setOptions = function (queueName, options) {
     options.queueName = queueName;
   }
 
-  extend(options, { 
+  extend(options, {
     assertQueue: this.assertQueuesOnFirstSend,
-    bus: this, 
+    bus: this,
+    confirmChannel: this.confirmChannel,
     correlator: this.correlator,
     formatter: this.formatter,
-    listenChannel: this.listenChannel, 
-    log: this.log, 
+    listenChannel: this.listenChannel,
+    log: this.log,
     queuesFile: this.queuesFile,
     sendChannel: this.sendChannel
   });
 };
 
-RabbitMQBus.prototype.send = function send (queueName, message, options) {
+RabbitMQBus.prototype.send = function send (queueName, message, options, cb) {
   options = options || {};
 
   if ( ! this.initialized) {
-    return this.on('ready', send.bind(this, queueName, message, options));
+    return this.on('ready', send.bind(this, queueName, message, options, cb));
   }
 
   this.setOptions(queueName, options);
-  if (this.queues[options.queueName] === undefined) {
-    this.queues[options.queueName] = new Queue(options);
+
+  var key = cb ? options.queueName + '.confirm' : options.queueName;
```

Comment:
Can you check for options.enableConfirms && cb here? We shouldn't name the queue .confirm if it ends up not being one because options.enableConfirms is not specified.